packages/superinput/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@botpress/superinput",
+  "version": "0.1.0",
+  "private": true,
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "dependencies": {
+    "@codemirror/autocomplete": "^0.19.9",
+    "@codemirror/closebrackets": "^0.19.0",
+    "@codemirror/commands": "^0.19.5",
+    "@codemirror/highlight": "^0.19.6",
+    "@codemirror/history": "^0.19.0",
+    "@codemirror/lang-javascript": "^0.19.3",
+    "@codemirror/language": "^0.19.6",
+    "@codemirror/rangeset": "^0.19.2",
+    "@codemirror/state": "^0.19.6",
+    "@codemirror/text": "^0.19.5",
+    "@codemirror/tooltip": "^0.19.10",
+    "@codemirror/view": "^0.19.21",
+    "@lezer/common": "^0.15.9",
+    "@lezer/markdown": "^0.15.2",
+    "espree": "^9.3.0",
+    "fast-memoize": "^2.5.2",
+    "react-json-view": "^1.21.3",
+    "tree-model": "^1.0.7",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "build": "yarn doctree && rollup -c",
+    "dev": "rollup -c -w",
+    "demo": "parcel demo/index.html",
+    "doctree": "ts-node --skip-project ./scripts/genDocTree.ts"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "peerDependencies": {
+    "@blueprintjs/core": "*",
+    "core-js": "*",
+    "react": "*",
+    "react-dom": "*"
+  },
+  "devDependencies": {
+    "@parcel/transformer-sass": "^2.2.1",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "@rollup/plugin-typescript": "^8.3.0",
+    "@types/lodash": "^4.14.177",
+    "@types/node": "^17.0.9",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@types/styled-components": "^5.1.15",
+    "glob": "^7.2.0",
+    "parcel": "^2.2.1",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
+    "rollup": "^2.64.0",
+    "rollup-plugin-dts": "^4.1.0",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-terser": "^7.0.2",
+    "ts-node": "^10.4.0",
+    "tslib": "^2.3.1",
+    "typescript": "^4.1.2",
+    "typescript-json-schema": "^0.53.0"
```

Comment:
We'll setup typescript as a yarn sdk pretty soon, so we'll remove that eventually but it's not blocking to merge this PR.